doc/arc42/src/04_solution_strategy.adoc

```diff
@@ -22,3 +20,31 @@ Motivate what you have decided and why you decided that way,
 based upon your problem statement, the quality goals and key constraints.
 Refer to details in the following sections.
 ****
+
+== Solution Strategy
+
+=== Security
+
+Since we're operating on very sensitive information, security is an important aspect of this project.
+
+==== Authentication
+
+In a later stage of the product, we need end-to-end encryption. As an example it needs to be possible to send encrypted data from one VE (A) to an other (B) and only B should be able to read (decrypt) the data. Therefore, we decided to use asymmetric encryption to secure the data. This means that we won't implement a classic authentication in the beginning. We simply make sure that only the correct participant can read the data.
+
+==== Asymmetric Encryption
+
+To encrypt the data asymmetrically, we're using the **RSA** algorithm. Each participant of the OpenPrevo system needs to have a private and public-key pair.
+
+The public key will be stored on the OpenPrevo Hub and can be distributed to all participants. The private key must not leave a VEs environment.
+
+NOTE: Depending on the amount of data we need to encrypt, we might have to use symmetrical encryption for the data and exchange the symmetric key via RSA. This will be decided depending on the attributes to use for the matching.
+
+==== Public Key Registry
+
+Every VE needs to register its public key on our OpenPrevo Hub. So the Hub will manage all public keys of every VE.
+
+==== Hashing
+
+According to data protection guidelines in Switzerland we are not allowed to read any personal data of an insured person on our platform. Therefore, a VE is responsible for sending us only a _hash_ of their data.
+
+The hash function we're going to use will be proposed by the data protection officer of the baloise insurance.
```

Comment:
For now I'd remove this sentence / line.

Reply:
removed the line and moved the chapter as discussed to the cross-cutting concepts.